src/bpf/bpfd-manager.h

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#pragma once
+
+#include "sd-bus.h"
+#include "sd-event.h"
+
+#include "hashmap.h"
+#include "macro.h"
+#include "varlink.h"
+
+/* Polling interval for BPF timer  */
+#define BPF_TIMER_INTERVAL_USEC 60 * USEC_PER_SEC /* 1 minute */
```

Comment:
please enclose in (), so that it's safe to use in arbitrary mathematical expressions